docs/guides/gateway/gateway_general_documentation.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Gateway General Documentation
+sidebar_position: 0
+---
+
+# Gateway General Documentation
+
+The Shifu project's gateway is a multi-protocol bridge that supports various protocols such as HTTP, MQTT and LwM2M. It provides a unified interaction interface for different IoT devices and systems, simplifying protocol conversion and integration.
+
+## Overview
+
+The **Shifu** gateway supports multiple communication protocols, including HTTP, MQTT, and LwM2M. It offers a universal solution for IoT ecosystems, allowing devices using different protocols to be managed and controlled through a single gateway, reducing development complexity and promoting integration with cloud services and other applications.
+
+The gateway component enables devices to adapt to a unified protocol and handle requests from the same server. It facilitates telemetry services to extract data from devices and supports data publishing from the cloud to devices. Through the same server, data can also be uniformly pushed to the cloud.
+
+## Key Features
+
+- **Protocol Conversion:** Supports converting HTTP, MQTT, and OPC UA requests into a unified internal message format, allowing developers to interact with different types of IoT devices without dealing directly with the underlying protocol details.
+- **Device Management:** Provides core functionalities such as device registration, reading attributes, executing commands, and monitoring device status, applicable to devices under multiple protocols.
+
+## Typical Use Case
+
+A typical use case is an HTTP or MQTT client interacting with IoT devices:
```

Comment:
need to think again for such use cases, it typically is asking why we need gateway.